crates/astria-core/src/protocol/genesis/v1alpha1.rs

```diff
@@ -131,6 +131,11 @@ impl Protobuf for GenesisAppState {
     type Error = GenesisAppStateError;
     type Raw = raw::GenesisAppState;
 
+    #[expect(
+        clippy::allow_attributes,
+        clippy::allow_attributes_without_reason,
+        reason = "false positive"
```

Comment:
Can you spell why it's a false positive?

Also, can you make a tracking issue for us to fix this once its fixed upstream?

Reply:
https://github.com/astriaorg/astria/issues/1580